index.html

```diff
@@ -0,0 +1,981 @@
+              </p>
+            </article>
+
+            <div class="bottom-social">
+              <button type="button" class="btn-comments">
+                READ 615 COMMENTS
+              </button>
+              <div class="social-icons-row">
+                <button type="button" class="fb"></button>
+                <button type="button" class="whatsapp"></button>
+                <button type="button" class="twitter"></button>
+                <button type="button" class="mail"></button>
+                <button type="button" class="share"></button>
+              </div>
+            </div>
+          </article>
+        </div>
+      </div>
+
+      <div class="sidebar">
+        <div>
+          <aside class="editor">Editors' pick</aside>
+        </div>
+        <div class="editor-pick">
+          <img
+            src="/images/editors_pick_1.png"
+            alt="thumbnail"
+            height="80"
+            width="80"
+          />
+
+          <div class="editor-pick-link">
+            <a
+              href="https://www.nytimes.com/2021/03/18/style/ludwig-ahgren-twitch-livestream.html?action=click&algo=lda_unique&block=editors_picks_recirc&fellback=false&imp_id=711436736&impression_id=21af6cd0-8b88-11eb-bd74-93281795ed16&index=0&pgtype=Article&region=ccolumn&req_id=139413807&surface=home-featured&variant=2_lda_unique"
+              >The Endless Stream</a
+            >
+          </div>
+        </div>
+        <div class="editor-pick">
+          <img
+            src="/images/editors_pick_2.jpg"
+            alt="thumbnail"
+            height="80"
+            width="80"
+          />
+
+          <div class="editor-pick-link">
+            <a
+              href="https://www.nytimes.com/2021/03/19/nyregion/andrew-cuomo-sexual-harassment-.html?action=click&algo=lda_unique&block=editors_picks_recirc&fellback=false&imp_id=62277491&impression_id=21af6cd1-8b88-11eb-bd74-93281795ed16&index=1&pgtype=Article&region=ccolumn&req_id=139413807&surface=home-featured&variant=2_lda_unique"
+            >
+              30 Years After Anita Hill, We Should All Know What Sexual
+              Harassment Is</a
+            >
+          </div>
+        </div>
+        <div class="editor-pick">
+          <img
+            src="/images/editors_pick_3.jpg"
+            alt="thumbnail"
+            height="80"
+            width="80"
+          />
+
+          <div class="editor-pick-link">
+            <a
+              href="https://www.nytimes.com/2021/03/17/books/review/this-is-the-fire-don-lemon.html?action=click&algo=lda_unique&block=editors_picks_recirc&fellback=false&imp_id=163566920&impression_id=21af6cd2-8b88-11eb-bd74-93281795ed16&index=2&pgtype=Article&region=ccolumn&req_id=139413807&surface=home-featured&variant=2_lda_unique"
+            >
+              Don Lemon’s New Book Hopes to Guide America Through a Conversation
+              About Race</a
+            >
+          </div>
+        </div>
+      </div>
```

Comment:
### Implementation details
- [ ] Mind to use `<aside>` tag for  this `Editor's picks` section? Inspiration can be drawn from this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/aside_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/).